rust/ql/test/query-tests/unusedentities/CONSISTENCY/DataFlowConsistency.expected

```diff
@@ -218,6 +218,9 @@ uniqueCallEnclosingCallable
 | unreachable.rs:294:13:294:32 | CallExpr | Call should have one enclosing callable but has 0. |
 | unreachable.rs:296:9:296:22 | CallExpr | Call should have one enclosing callable but has 0. |
 | unreachable.rs:301:13:301:32 | CallExpr | Call should have one enclosing callable but has 0. |
+| unreachable.rs:306:5:306:18 | CallExpr | Call should have one enclosing callable but has 0. |
+| unreachable.rs:308:5:308:18 | CallExpr | Call should have one enclosing callable but has 0. |
+| unreachable.rs:310:5:310:18 | CallExpr | Call should have one enclosing callable but has 0. |
```

Comment:
These added inconsistencies are only because more nodes are now included in the CFG. Before the CFG would end in a loop in the `for_ever` async block. Now the block gets its own scope and the remaining code are the block is also in the CFG.